### PR TITLE
Markers

### LINKED ISSUE #24: Show markers for search results
Search for any place and results should be indicated by makers

### PR DESCRIPTION
**Pull request by ivinjose**
This fixes #24

### CODE REVIEW

File: app/scripts/ui/Map.js [hunk 1]
```diff
@@ -2,29 +2,64 @@
...
         });
     },
+    componentDidMount: function(){
+        this._getGeocode(this.props.zipcode).then( this._setMapCenter ).then( this._showPlaces );
+    },
+    componentWillReceiveProps: function(nextProps){
+        this._getGeocode(nextProps.zipcode).then( this._setMapCenter ).then( this._showPlaces );
+    },
+    shouldComponentUpdate: function(nextProps, nextState){
+        return nextProps.zipcode!=this.props.zipcode;
```
Comment (shidhincr) [inline app/scripts/ui/Map.js]: I would prefer !==,  else we can see JSHint crying

File: app/scripts/views/search.js [hunk 2]
```diff
@@ -10,7 +10,9 @@ var Col     = require('react-bootstrap/Col');
 
 var Search  = React.createClass({
     render: function(){
-        var zipcode = location.href.split(/search\//)[1].match(/\d+/g)[0];
+        var query   = location.href.split(/search\//)[1].split('/');
```
Comment (shidhincr) [inline app/scripts/views/search.js]: I guess there is another way to retrieve the query from the Router component itself. Can you check their documentations?

Comment (ivinjose) [inline app/scripts/views/search.js]: Updated the PR to use rect-router's Location object

File: app/scripts/ui/Map.js (hunk 1 above)
Comment (ivinjose) [inline app/scripts/ui/Map.js]: Done. No more teary JSHint. :)

**Comment by shidhincr** [comment]
sure

**Pull request merged**